references/dev_getting_started.md

```diff
@@ -0,0 +1,104 @@
+# Getting started for development on this project
```

Comment:
We can actually remove this from the stable branch that we'll push to for the report